app/logic/Abstract/Account.ts

```diff
@@ -110,6 +110,12 @@ export class Account extends Observable {
   /** Saves the config in this account to disk.
    * Does not save the contents, e.g. messages. */
   async save(): Promise<void> {
+    throw new AbstractFunction();
+  }
+
+  /** Saves the config in this account and its dependent accounts to disk. */
+  async saveAll(): Promise<void> {
+    await this.save(); // must be first, because of foreign key constraint
     for (let dependent of this.dependentAccounts()) {
       await dependent.save();
```

Comment:
```suggestion
      await dependent.saveAll();
```

Bah, at one point I had this change locally, but I couldn't decide whether or not `saveAll` was a good idea, and removed the change, only to put `saveAll` back, but without the change. Sigh...

Reply:
Are you saying that it's broken as landed?

Reply:
If so, can you please make a PR to fix it?

Reply:
It doesn't do what the old code does, which is to recursively save all dependent accounts. (We don't actually have more than one nesting of account, especially not at the point of the only caller, so it's not broken in that sense.)

Reply:
Oh, I see, you're thinking of multiple levels of dependent accounts.
Yes, it would be a nice change to make, but I agree: We don't have that case right now.